tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -77,12 +82,13 @@ def _validate_settings(self, settings: Dict[int, int]) -> None:
         if self._datagram_setting is None:
             raise SettingsError("HTTP Datagrams support required")
 
-
     def _get_local_settings(self) -> Dict[int, int]:
         settings = super()._get_local_settings()
         settings[H3DatagramSetting.RFC] = 1
         settings[H3DatagramSetting.DRAFT04] = 1
         settings[H3ConnectionWithDatagram.ENABLE_CONNECT_PROTOCOL] = 1
+        # This connection can handle only one WebTransport session.
```

Comment:
IIUC, this is a limitation of not implementing pooling, so maybe we should remove this comment?

Reply:
Yes, the server doesn't support pooling. It also cannot handle multiple WebTransport sessions at the moment. The comment is to describe the situation why we use 1 here. Happy to remove the comment, but can you elaborate on why we should remove the comment?

Reply:
Ah okay, I thought it was a client-side limitation. Since it's a server side-side limitation as well, leaving the comment is okay :)